cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -337,7 +337,17 @@ Usage: cardano-cli conway governance delegation-certificate
                                                               ( --stake-pool-verification-key STRING
                                                               | --cold-verification-key-file FILE
                                                               | --stake-pool-id STAKE_POOL_ID
-                                                              | --dummy-drep-option ARG
+                                                              | --drep-verification-key STRING
+                                                              | --drep-verification-key-file FILE
+                                                              | --drep-key-hash HASH
+                                                              | ( --drep-verification-key STRING
```

Comment:
Something's wrong with this parser - it's duplicated.

Reply:
It's duplicated because on of the options allows you to delegate voting stake and reward stake in a single delegation certificate

Reply:
This is because one of the options allows you to delegate voting stake and reward stake in a single certificate

Reply:
Oh I see it now. Makes sense I think, but it's a little confusing at the first glance. The duplicated help text also doesn't clarify that.

Reply:
It's definitely confusing. We need to make this clearer to the user but we can do this after sanchonet.